integration/aggregate_documents_compat_test.go

```diff
@@ -92,6 +95,12 @@ func testAggregateStagesCompatWithProviders(t *testing.T, providers shareddata.P
 				pipeline = append(pipeline, bson.D{{"$sort", bson.D{{"_id", 1}}}})
 			}
 
+			opts := new(options.AggregateOptions)
```

Comment:
I think we should use `options.Aggregate()` like we do with `options.FindOne()`, `options.Update()`, `options.Delete()`, etc. Currently, there are two outliers: `options.AggregateOptions` and `options.IndexOptions`

Reply:
I will standadise options.AggregateOptions and options.IndexOptions to use options.Aggregate() and  options.Index()